### ISSUE TITLE
Can't select Config or Maintenance

### ISSUE THREAD

**Issue by mostlyflawles** [Dijker/nu.dijker.PaperTrails#44]
Homey 5.0.4
PaperTrails 0.6.3

In the app it says "Download doesn't work on your mobile... check https://developer.athom.com/tools/app-settings".
There I go to PaperTrails app but I can't press on Config or Maintenance, only Log is selected.
I tried Edge, Firefox and Chrome.

**Comment by Dijker** [comment]
Yep this is a know issue on the Developer site, on mobile it works OK for Config buy Maintenance does not from FW version 2.0 and above.   for that part go to the https://homeybackup.web.app/login/
